start_esp/start_esp.py

```diff
@@ -238,6 +238,11 @@ def fetch_service_config(args):
     try:
         # Get the access token
         if args.service_account_key is None:
+            if args.check_metadata == False:
```

Comment:
this may have problem.   for GKE, we want to fetch access token from metadata server by default

Reply:
Removed checking args.check_metadata option